### ISSUE TITLE
Final Review

### ISSUE THREAD

**Issue by jmewong** [tinali0923/Volatility-Prediction#12]
Abstract was beautiful––I like that you guys took a non-traditional route to the problem. You guys considered many edge cases, and I was very impressed by that. For example, you guys normalized your data as well as used PCA to impute missing values.

The basic explorations visualizations look cool. I loved that you used visualizations to discover that PE is an important coefficient. I also liked that you made a visualization of the covariance matrix to predict relationships between features.

In section 3.1 about linear regression, the methodology was a little hard to follow and so were the reporting of the results. I don't understand what you meant by "For Huber, the regularizor is l2 norm." How did you decide to use the combination of huber loss and l2 regularizer? Why did you decide to use ridge regression? I would have like to see more explanation as to why you picked these methods i.e. was it the shape of your data? The need for sparsity? The belief in a linear relationship? etc. Also, it doesn't seem reasonable to compare loss functions using different regularizers. (For some loss functions, you used l2, but for one of them you used l1). Maybe it would be better to compare both types of regularizers separately. 

Finally, your postulation that "Judging from that this number didn’t increase too much from the mean squared error obtained in training set by Lasso, our model doesn’t have the problem of over fitting" was concerning to me because the mean squared error was enormous (~316), so it does not seem like the final model you used is appropriate. Or at least, I would like to see more explanation and more model explorations. However, you solved the overfitting issue by comparing this MSE with the test set MSE, which is great!

Overall, parameter exploration and data cleaning/fixing methodologies were fantastic. I loved reading the report. Great job!
